src/client.js

```diff
@@ -61,9 +66,17 @@ async function onRequest(req, res) {
         return onAPIRequest(req, res)
     }
 
-    logger.info(`${req.method} ${req.url}`)
+
 
     let urlp = url.parse(req.url)
+
+    if (urlp.pathname === '/proxy.pac') {
+        let PAC = pac.replace('127.0.0.1:8099',req.headers.host);
```

Comment:
Delay further review to this night.
Work time.😝

On Tuesday, 16 August 2016, agentmario <notifications@github.com> wrote:

> In src/client.js
> <https://github.com/yukixz/kcsp-client/pull/3#discussion_r74891773>:
>
> >
> >      let urlp = url.parse(req.url)
> > +
> > +    if (urlp.pathname === '/proxy.pac') {
> > +        let PAC = pac.replace('127.0.0.1:8099',req.headers.host);
>
> Change proxy addr if local port changed or connect from other devices.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/yukixz/kcsp-client/pull/3/files/c3ab4ebb4f85b5542c4c9b26395ebd32347dbb92#r74891773>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADLuwbF9YWahWtmR7dnZCKH-EwVaDjiaks5qgW-VgaJpZM4JlITg>
> .
>


-- 
*Dazzy Ding*
Email: h@dazzyd.org